charts/ccpay-bubble-frontend/values.yaml

```diff
@@ -1,11 +1,26 @@
 nodejs:
   applicationPort: 3000
+  ingressHost: ccpay-bubble.{{ .Values.global.environment }}.platform.hmcts.net
+  aadIdentityName: ccpay
+  memoryLimits: 1024Mi
+  cpuLimits: 1000m
+  readinessDelay: 30
+  livenessDelay: 30
+  keyVaults:
+    "ccpay":
+      resourceGroup: ccpay		
```

Comment:
this isn't needed
```suggestion
```

Reply:
Ok, I'll remove at the next PR as I have something else to clean up.